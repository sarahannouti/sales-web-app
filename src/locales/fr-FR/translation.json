{
  "hello": "Bonjour",
  "greeting": "Salutations",
  "hi": "Salut",
  "products": {
    "title": "Produits",
    "count_zero": "Pas de produit",
    "count_one": "1 produit",
    "count_other": "<1>{{count}}</1> produits"
  }
}
