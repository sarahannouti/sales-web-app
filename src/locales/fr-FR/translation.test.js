import translation from "./translation.json";

describe("fr-FR translation", () => {
  test("writes all translations", () => {
    expect(translation).toMatchSnapshot();
  });
});
