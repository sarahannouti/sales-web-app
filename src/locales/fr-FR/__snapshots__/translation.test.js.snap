// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`fr-FR translation > writes all translations 1`] = `
{
  "greeting": "Salutations",
  "hello": "Bonjour",
  "hi": "Salut",
  "products": {
    "count_one": "1 produit",
    "count_other": "<1>{{count}}</1> produits",
    "count_zero": "Pas de produit",
    "title": "Produits",
  },
}
`;
